ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -83,6 +99,61 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
+  renderIncomingTransactionsOptIn() {
```

Comment:
I had to make the same arrangement for renderXXX methods here like I did in advanced tab components.